server/plugins/renderer/renderer.module.js

```diff
@@ -390,7 +390,7 @@ internals.getPencilResponse = function (data, request, response, configuration)
     data.context.settings.theme_version_id = Utils.int2uuid(1);
     data.context.settings.theme_config_id = Utils.int2uuid(request.app.themeConfig.variationIndex + 1);
     data.context.settings.theme_session_id = null;
-    data.context.settings.maintenance = {secure_path: `http://localhost:${internals.options.stencilEditorPort}`};
+    data.context.settings.maintenance = {secure_path: `http://localhost:8181`};
```

Comment:
just hardcoding the port here since that was the default if the option for the port was not being passed.

Reply:
What is this used for? And if it's still useful, why not use `stencilServerPort` rather than the `stencilEditorPort` (which is no longer used)?